delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/projects/ProjectsImpl.scala

```diff
@@ -101,15 +130,17 @@ final class ProjectsImpl private (
       organizations.fetchActiveOrganization(ref.organization).void
     ) >>
       fetch(ref).flatMap {
-        case resource if options.contains(ProjectFetchOptions.NotDeprecated) && resource.deprecated =>
+        case resource if options.contains(ProjectFetchOptions.NotDeleted) && resource.value.markedForDeletion =>
```

Comment:
Shouldn't we do it the other way around ?
If you don't specify that you want to include deleted projects, it will always fail ?
If somebody fetches a resource which the project has been marked as deleted, it should fail no ?
As otherwise it could revive the persistent actor

Reply:
If someone fetches a project that has been marked for deletion but didn't specify in the options `ProjectFetchOptions.NotDeleted` it should go on (same logic as for deprecated). This sort of fetch doesn't happen very often in our system, but it can occur.

Reply:
But if someone fetches a resource while deletion is happening, it can cause problems